runtime/gc_base/GCExtensions.hpp

```diff
@@ -261,12 +262,23 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	static MM_GCExtensions* getExtensions(MM_EnvironmentBase* env) { return getExtensions(env->getExtensions()); }
 
 	MMINLINE J9JavaVM* getJavaVM() {return static_cast<J9JavaVM*>(_omrVM->_language_vm);}
+	
+	/**
+	 * Return ownable synchronizer object lists by first ensuring that the lists are in a consistent state (e.g., during concurrent gc).
+	 * This should be used by any external consumer (non-GC consuming the list)     
+	 * @param vmThread The current J9VMThread thread (used to invoke j9gc apis if required)
+	 * @return Linked list of ownable synchronizer objects
+	 */
+	MM_OwnableSynchronizerObjectList* externalGetOwnableSynchronizerObjectLists(J9VMThread *vmThread);
```

Comment:
i'd put 'external' to the end. it's less english, but that's more in line with some other GC method names.